### PR TITLE
Rename _vagrant to vagrant

### PR DESCRIPTION
**Pull request by gianarb**
Apparently the idea to prefix a package with an underscore is not that
smart as I thought. Yes `go test` does not run it by default when you
run `go test ./...` but also other commands like `go mod tidy` do not
work consistently.

Nothing changes in practice. By default only unit tests run. Setting the
new environment variable: `TEST_WITH_VAGRANT` you include the test who
uses vagrant.

### CODE REVIEW

File: test/_vagrant/vagrant_test.go
Comment (mmlb) [inline test/_vagrant/vagrant_test.go:1]: did you forget to commit the new dir?

Comment (gianarb) [reply test/_vagrant/vagrant_test.go:1]: did I?! Maybe!!!

Comment (gianarb) [reply test/_vagrant/vagrant_test.go]: Fixed!! :D Thanks

File: test/vagrant/vagrant_test.go
Comment (mmlb) [inline test/vagrant/vagrant_test.go:20]: can also add a build constraint right? I believe that works for test files too. Not sure if thats any better than this env look up though tbh.

**Review by mmlb — APPROVED** [review]

**Comment by mmlb** [comment]
@gianarb I imagine you want to run the vagrant action in this PR too right?

**Comment by gianarb** [comment]
yes but for some reason the runner is not up, so I am looking at it first. But yes! I will merge only when vagrant on packet workflow is happy as well

**Comment by mmlb** [comment]
Hmm I added the label to see if GH will then block the PR until the runner is back and tests pass.

**Comment by mmlb** [comment]
Yep looks like it does, now you can't hit merge by mistake :p

**Comment by mmlb** [comment]
Also curious to see how this ends up failing.

**Comment by gianarb** [comment]
I am merting it manually because Mergify can't merge PRs who changes workflows

**Pull request merged**
